src/Infrastructure/Serialization/Symfony/Normalizer/EnumNormalizer.php

```
@@ -1,43 +1,25 @@
 
-    /** {@inheritdoc} */
-    public function supportsNormalization($data, $format = null)
+    public function supportsNormalization(mixed $data, ?string $format = null, array $context = []): bool
     {
-        return $data instanceof Enum;
+        return $data instanceof \BackedEnum;
     }
 
-    /** {@inheritdoc} */
-    public function normalize($object, $format = null, array $context = [])
+    public function normalize(mixed $object, ?string $format = null, array $context = []): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$format'. (undefined)

(UnusedFormalParameter)